z_concepts.md

```diff
@@ -15,7 +15,7 @@ This section discusses the following main topics:
 
 ## GitHub
 
-Essentially, there are two main versions of GitHub the UNIX/Command Line version and the online interface version. Though a GitHub can be thought of as a repository for hosting content, Git itself is a command-line tool that power users (such as engineering) use. While I have explored using the CLI and [PyCharm](https://www.jetbrains.com/pycharm), etc., personally, I prefer the interface, which is more user-friendly. See [Git Wiki](https://en.wikipedia.org/wiki/Git) for more details. 
```

Comment:
Line 18 - Suggested Removals = @techuxwriter I think that words such as power user is a bit outdated. I'd suggest removing this. Some other minor suggestions on this line too.